packages/foreman/keycloak-httpd-client-install/keycloak-httpd-client-install.spec

```
@@ -2,18 +2,29 @@
 %global pypi_name keycloak-httpd-client-install
 %global summary Tools to configure Apache HTTPD as Keycloak client
 
+%if 0%{?rhel} == 7
+%global with_python2 1
+%else
+%global with_python3 1
+%endif
```

```suggestion
%if 0%{?rhel} == 7
%bcond_without python2
%bcond_with python3
%else
%bcond_with python2
%bcond_without python3
%endif
```

`bcond_with` means "it defaults to without", and `bcond_without` "it defaults to with", confusing, but that are the rules